gax/BUILD.bazel

```diff
@@ -16,29 +16,42 @@ package(default_visibility = ["//visibility:public"])
 
 licenses(["notice"])  # Apache 2.0
 
+# Note: googleapis doesn't define a cc_proto_library for operations_proto,
```

Comment:
Off-topic: I think it does not define a `cc_proto_library()` for anything, do we have plans to create a repository with `cc_proto_library()` for all the APIs we need?

Reply:
I think @vam-google can provide more info on that subject.